AppCenterLoaderApp/appcenter-loader/src/main/java/com/microsoft/appcenter/loader/AppCenterLoader.java

```diff
@@ -96,7 +96,7 @@ public boolean onCreate() {
             isModuleAvailable("com.microsoft.appcenter.push.Push", "Push")) {
             Push.setListener(new UnityAppCenterPushDelegate());
             classes.add(Push.class);
-            Push.setSenderId(getStringResource(SENDER_ID_KEY));
+            //Push.setSenderId(getStringResource(SENDER_ID_KEY)); // deprecated method
```

Comment:
This puts pushes in non-working state without any warning. There are a few ways to deal with it at the moment:
- Revert to see the warning;
- Comment the whole block here;
- Migrate to new version;